io.github.uowuo.abaddon.yml

```diff
@@ -0,0 +1,158 @@
+app-id: io.github.uowuo.abaddon
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: abaddon
+finish-args:
+  # Wayland access
+  - --socket=wayland
+  # X11 fallback
+  - --socket=fallback-x11
+  - --share=ipc #x11 depends on this
+  # Needs network
+  - --share=network
+  # Needs sound + mic
+  - --socket=pulseaudio
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/lib/
+  - --env=LD_LIBRARY_PATH=/app/lib/lib64/
+  # Prevents freeze when closing while connected.
+  #- --socket=session-bus
+  - --talk-name=org.gtk.vfs.*
+  - --own-name=com.github.uowuo.abaddon
+
+modules:
+  - name: abaddon
```

Comment:
this is the main module, it should be last as it is "build" after the dependencies....